students/junghyun/users/views.py

```diff
@@ -1,3 +1,34 @@
+import json
+import re
+from sre_constants import SUCCESS
+
 from django.shortcuts import render
+from django.http  import JsonResponse
+from django.views import View
+
+from .models import User
 
-# Create your views here.
+class UsersView(View):
```

Comment:
작성해주신 `class`는 `회원가입을 위한 View`이기 때문에 그에 맞게 네이밍을 해주세요!
```suggestion
class SignUpView(View):
```